docs/security.md

```diff
@@ -253,7 +260,44 @@ sasl.jaas.config=org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginMo
    oauth.audience="https://broker.example.com";
 ```
 
+### Together with Pulsar's authentication
+
+Since KoP reuses Pulsar's authentication providers for authentication, it can be enabled with other authentication providers at the same time.
```

Comment:
```suggestion
Since KoP reuses Pulsar's authentication providers for authentication, you can KoP's authentication providers and Pulsar authentication providers at the same time.
```
```suggestion
Since KoP reuses Pulsar's authentication providers for authentication, it can be enabled with other authentication providers at the same time.
```